### ISSUE TITLE
Nest-Cam On/Off switch not working

### ISSUE THREAD

**Issue by leoneleone** [KhaosT/homebridge-nest-cam#23]
I’ve noticed that the On/Off switch that the plugin creates does not work for me. Also, ever since adding homebridge-nest-cam to my homebridge setup, Nest Home/Away does not properly.

I’d like to point out I have both homebridge-nest and homebridge-nest-cam running to separate Raspberry Pi(s).

**Comment by djglisson2** [comment]
Does this plugin have a on/off switch for the camera? If so, I'm not even seeing it...

**Comment by KhaosT** [comment]
Nope, HomeKit itself doesn’t have the idea of turning off a camera.

**Comment by QandnotU** [comment]
@KhaosT is possible to expose a switch that taps into this: https://developer.apple.com/documentation/homekit/hmcamerastreamcontrol
